### PR TITLE
Crabify

### PR DESCRIPTION
**Pull request by Lonami**
# 🦀

The change that prompted me to do this is https://github.com/LonamiWebs/Telethon/commit/4d3ff0e175da6db909794ab66d2d8fa6ed482dad. I am pleasantly surprised at how easy both PyO3 and `setuptools_rust` are. We should setup CI to build in as many Python versions as possible, and automatically publish the binary wheels on tag.

This should be faster than tiny-aes, because the Rust code does use AES-NI when possible.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,7 +39,7 @@ def main(args):
             "Intended Audience :: Developers",
             "Topic :: Security :: Cryptography",
 
-            "License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
+            'License :: OSI Approved :: MIT License',
```
Comment (cher-nov) [inline setup.py:42]: MIT requires to provide copy of the license to be shared with every instance of the application, which is inconvient.
This is the reason why I chose CC0 (public domain) back then.

Comment (Lonami) [reply setup.py:42]: Neither `grammers-cryptg` or `pyo3` are CC0, though. Can we still use CC0 if we depend on non-CC0 libraries?

Comment (cher-nov) [reply setup.py:42]: > Can we still use CC0 if we depend on non-CC0 libraries?

Yes, as long as you share licenses for the works being used.
https://wheel.readthedocs.io/en/stable/user_guide.html#including-license-files-in-the-generated-wheel-file

Also please remove `build_ffi.py` and other stuff related to current approach based on `cffi`.

Comment (Lonami) [reply setup.py:42]: > Also please remove `build_ffi.py` and other stuff related to current approach based on `cffi`.

Should I get rid of `appveyor.yml` and `.travis.yml` as well?

Comment (cher-nov) [reply setup.py]: > Should I get rid of `appveyor.yml` and `.travis.yml` as well?

Have you provided another way to automatically build wheels for all supported flavours of Windows / Linux / macOS?

Comment (Lonami) [reply setup.py]: Oh, no, forgot Linux. I'll get rid of the GH Actions script, and we'll have to modify travis one, I have no idea how it works.

Comment (cher-nov) [reply setup.py]: Also please restore proper string punctuators on this line.
`"` means human-readable text, while `'` means identifier-like technical strings.

**Comment by Lonami** [comment]
(The CI script is untested)

**Comment by cher-nov** [comment]
> 
> 
> (The CI script is untested)

Please supply these options in the `env:` section:
```
CIBW_SKIP: ?p27-*
CIBW_BUILD_VERBOSITY: 3
BUILD_OUTPUT_PATH: wheelhouse
```

`BUILD_OUTPUT_PATH` then must be substituted in call to `cibuildwheel` and wheel output path.

**Comment by Lonami** [comment]
Note for future self: cross-check the CI configuration file with https://github.com/PyO3/setuptools-rust/blob/main/examples/html-py-ever/.github/workflows/upload.yml.

**Pull request merged**